src/routes/setup/+page.svelte

```diff
@@ -1,104 +1,117 @@
+  import { derived, get } from 'svelte/store';
+  import { HttpHandler } from '../../data/httpHandler';
+  import type { Parameters_Parameter } from '../../data/unity-cs-manager/protobuf/extensions';
+  import InputField from '../../components/InputField.svelte';
+  import SelectField from '../../components/SelectField.svelte';
+  import OptionalParametersAccordion from '../../components/OptionalParametersAccordion.svelte';
 
   let httpHandler = new HttpHandler();
 
   let parameters = get(parametersStore); // Get the current value of the store
 
-  let key = "";
-  let value = "";
+  let key = '';
+  let value = '';
   let list: { key: string; value: string }[] = [];
 
-  const venueAndProjectStore = derived([venueStore, projectStore], ([$venueStore, $projectStore]) => {
-    // Do validations and derivations here
-    return { venueIsValid: /^[a-z0-9]+$/i.test($venueStore), projectIsValid: /^[a-z0-9]+$/i.test($projectStore) };
-  });
+  const venueAndProjectStore = derived(
+    [venueStore, projectStore],
+    ([$venueStore, $projectStore]) => {
+      // Do validations and derivations here
+      return {
+        venueIsValid: /^[a-z0-9]+$/i.test($venueStore),
+        projectIsValid: /^[a-z0-9]+$/i.test($projectStore)
+      };
+    }
+  );
 
   function handleInputChange(e: CustomEvent) {
     const target = e.detail.target as HTMLInputElement;
     const { id, value } = target;
-    if (id === "project") projectStore.set(value);
-    if (id === "venue") venueStore.set(value);
+    if (id === 'project') projectStore.set(value);
+    if (id === 'venue') venueStore.set(value);
     // Handle other changes
   }
 
   function handleSubmit() {
-    console.log("project: " + get(projectStore));
-    console.log("venue: " + get(venueStore));
     const unsubscribe = parametersStore.subscribe((items) => {
       let l = items.parameterlist;
-      l["project"] = createBaseParameters_Parameter({
-        name: "project",
-        type: "String",
+      l['project'] = createBaseParameters_Parameter({
+        name: 'project',
+        type: 'String',
         insync: true,
         value: get(projectStore),
         tracked: true
       });
-      l["venue"] = createBaseParameters_Parameter({
-        name: "venue",
-        type: "String",
+      l['venue'] = createBaseParameters_Parameter({
+        name: 'venue',
+        type: 'String',
         insync: true,
         value: get(venueStore),
         tracked: true
       });
       installRunning.set(true);
       installError.set(false);
-      console.log(items.parameterlist);
       httpHandler.updateParameters(items.parameterlist);
     });
     unsubscribe();
-
   }
 
   parametersStore.subscribe((value) => {
     parameters = value; // Update parameters whenever the store changes
 
-    const venuekey = "/unity/core/venue";
-    const projectkey = "/unity/core/project";
+    const venuekey = '/unity/core/venue';
+    const projectkey = '/unity/core/project';
```

Comment:
@jdrodjpl I'm wondering why we would still need these keys, as these SSM entries should be going away completely.  Maybe there is a reason I'm not understanding?